usagov/usagov-2021
# Usagov 1722 make usagov logo responsive

## Jira Task


https://cm-jira.usa.gov/browse/USAGOV-1722-

## Description

Added srcset and responsive sizing to USAgov logo in the <img> tag in top_nav.html.twig

## Type of Changes

- [x ] Frontend (Twig, Sass, JS)
  - Add screenshot showing what it should look like
  
<img width="344" alt="image" src="https://github.com/user-attachments/assets/cce77c47-5a5e-44d9-a8f3-7643a8a4ad97">
<img width="229" alt="image" src="https://github.com/user-attachments/assets/1e4546eb-9ada-4544-9c63-e01f3695f0a0">


## Testing Instructions



Run localhost, resize the window to be big and small and make sure the usagov logo acts in the same responsive manner as prod. 

check lighthouse > run lighthouse audit > make sure "Properly Size Image" and "Explicit width and height for images" does NOT appear in the failed audits.

### Change Requirements



### Validation Steps

- Test instruction 1
- Test instruction 2
- Test instruction 3

## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
